tests/integration/tier0/resolve-broken-ol8-rollback/main.fmf

```diff
@@ -1,13 +1,6 @@
 summary: inhibit-if-oracle-system-uses-not-standard-kernel
 
 tier: 0
-adjust:
-  enabled: false
-  when: >
-    distro != centos-7 and
-    distro != centos-8 and
-    distro != oracle-7 and
-    distro != oracle-8
 
 /good:
```

Comment:
Can we remove this sub test, since there is only one test case?

Reply:
Good catch, I removed it.